cmd/podman/network_create.go

```diff
@@ -0,0 +1,69 @@
+// +build !remoteclient
+
+package main
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/containers/libpod/cmd/podman/cliconfig"
+	"github.com/containers/libpod/pkg/adapter"
+	"github.com/containers/libpod/pkg/rootless"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	networkcreateCommand     cliconfig.NetworkCreateValues
```

Comment:
Nit: `s/networkcreate/networkCreate` to be consistent with the naming in `cliconfig`.

Reply:
This isn't fixed?

Reply:
fixed now